.storybook/config.js

```diff
@@ -39,7 +39,11 @@ addParameters({
 
 addDecorator(story => (
   <Fragment>
-    <GlobalStyles fonts={Object.values(fontFaces)} />
+    <GlobalStyles
```

Comment:
This file cant be changed in this PR. Need to reset this file `git checkout latest .storybook/config.js` and redo this once this PR is merged and `psammead-styles` is published and pulled into simorgh :)

Reply:
NB: for anyone curious, this is because storybook uses published dependancies like
```
import * as fontFaces from '@bbc/psammead-styles/fonts';
```
So need to wait for it to be published to work :)

Reply:
Will fix after this gets merged